src/components/styled/index.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+import { Form } from 'lattice-fabricate';
+
+const EdgelessForm = styled(Form)`
+  margin: -30px;
+`;
```

Comment:
No change required, just a thought. Do you HAVE to do `styled(Form)`? Wouldn't the same thing work if it was just a wrapper?

Reply:
just seems easier to use a styled form